File: fibonacci/fibonacci.py
def fibonacci(n):
    # Implementation with stack overflow risk
    # Bug: Recursive implementation without tail-call optimization
    if n <= 0:
        return 0
    elif n == 1:
        return 1
    else:
        return fibonacci(n - 1) + fibonacci(n - 2)  # Will stack overflow on large n

if __name__ == "__main__":
    # This could cause stack overflow for large values
    print(fibonacci(35))

File: fibonacci/fibonacci.js
function fibonacci(n) {
    // Implementation with integer overflow issue
    if (n <= 0) return 0;
    if (n == 1) return 1;
    
    let prev = 0;
    let curr = 1;
    // Bug: No checks for integer overflow
    for (let i = 2; i <= n; i++) {
        let next = prev + curr;  // Will overflow for large n
        prev = curr;
        curr = next;
    }
    return curr;
}

// Test with a value that will cause integer overflow
console.log(fibonacci(77));